### DISCUSSION TITLE
Getting Copilot for free

### DISCUSSION THREAD

**Discussion by yachty66** [community/community#38991] [GitHub Education]
I have a subscription for Github Copilot and also have a GitHub student account. I want to know if Copilot is now automatically for free or if I still need to configure something. I can only read about that GitHub Copilot is free for students but actually cannot really find which instructions to follow for having Copilot for free.

**Comment by dhrubasaha08** [comment]
Just login with your account associated with GitHub Student Developers Pack !

**Comment by SilasRodrigues19** [comment]
@yachty66 Just go to [this link](https://github.com/settings/billing) and activate it. A message will appear that you are eligible, as in the screnshoot below.

![image](https://user-images.githubusercontent.com/47532703/201540302-080d6884-2074-445c-b773-70db2fb047d5.png)

**Comment by samus-aran** [comment]
Hi @yachty66 👋 

Where you able to get access to GitHub Copilot as a student?

**Reply by yachty66** [reply]
I was signed up to Copilot without student plan. When I had the student plan I was still paying. Now I cancelled it and will subscribe again when possible. This time as student. Thanks for the help.
